packages/wallet/backend/src/quote/controller.ts

```diff
@@ -11,12 +11,16 @@ interface QuoteControllerDependencies {
   quoteService: QuoteService
 }
 
+export type EnrichedQuote = Quote & {
+  fee?: Amount & { conversionRate: number }
```

Comment:
I do not think we need to send the conversion rate 🤔.

Reply:
ok, I removed it.